source/Ship.cpp

```diff
@@ -564,9 +549,12 @@ void Ship::FinishLoading(bool isNewInstance)
 	// Check that all the "equipped" weapons actually match what your ship
 	// has, and that they are truly weapons. Remove any excess weapons and
 	// warn if any non-weapon outfits are "installed" in a hardpoint.
+	auto equipped = armament.GetEquipped();
 	for(auto &it : equipped)
 	{
-		int excess = it.second - outfits[it.first];
+		auto outfitIt = outfits.find(it.first);
+		int nrOutfits = (outfitIt != outfits.end() ? outfitIt->second : 0);
```

Comment:
Maybe `amount`? Or something else. Using `nr` for number feels a bit weird imo.

Reply:
`amount` is fine for me.